docusaurus/docs/core-concepts/security/authorization/posture-checks.md

```diff
@@ -47,7 +47,7 @@ The following Posture Check types are currently defined:
 - [Multi Process](#multi-process) - requires a client be running one or more applications
 - [Windows Domain](#windows-domain) - requires the client be a member of a specific domain
 
-## OS/ OS Version
+### Operating System {#os-os-version}
```

Comment:
maybe this makes an anchor on the page? why not let docusaurus just generate what it wants?

Reply:
It creates a predictable URL fragment/anchor to reliably target a heading with a link. The two main reasons this is needed are when a heading has special characters or unusual whitespace that increase the likelihood of human error when writing the href that targets the fragment, and when there are ambiguous fragments because the same Markdown heading is used more than once. Docusaurus will auto-generate a serial number for each redundant heading's fragment in that scenario, which makes them unambiguous and unpredictable.

Reply:
https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md024